src/view/direct_mode/direct_scene_selector.py

```diff
@@ -0,0 +1,130 @@
+# coding=utf-8
+"""mange different scenes"""
+from typing import TYPE_CHECKING
+
+from PySide6 import QtWidgets, QtGui
+
+from model.broadcaster import Broadcaster
+from model.patching_universe import PatchingUniverse
+from view.direct_mode.universe_selector import UniverseSelector
+
+if TYPE_CHECKING:
+    from view.main_window import MainWindow
+
+
+class DirectSceneSelector(QtWidgets.QTabWidget):
+    """Widget to mange different scenes in Tab Widgets"""
+
+    def __init__(self, broadcaster: Broadcaster, parent: "MainWindow") -> None:
+        super().__init__(parent=parent)
+        self._broadcaster = broadcaster
+        self.setTabPosition(QtWidgets.QTabWidget.TabPosition.West)
+        self._last_tab: int = 0
+        self._scenes: list[UniverseSelector] = []
+        self.addTab(QtWidgets.QWidget(), "+")
+        self.currentChanged.connect(self._tab_changed)
+        self.tabBarClicked.connect(self._tab_clicked)
+        self._broadcaster.add_universe.connect(self.add_universe)
+
+        self._toolbar: list[QtGui.QAction] = []
+        save_button = QtGui.QAction("Save")
+        load_button = QtGui.QAction("Load")
+        save_button.triggered.connect(self._save_scenes)
+        load_button.triggered.connect(self._load_scenes)
+        self._toolbar.append(save_button)
+        self._toolbar.append(load_button)
+
+    @property
+    def toolbar(self) -> list[QtGui.QAction]:
+        """toolbar for patching_mode"""
+        return self._toolbar
+
+    def contextMenuEvent(self, event):
+        """context menu"""
+        for index in range(self.tabBar().count() - 1):
+            if self.tabBar().tabRect(index).contains(event.pos()):
+                menu = QtWidgets.QMenu(self)
+
+                rename_scene = QtGui.QAction('rename Scene', self)
+                delete_scene = QtGui.QAction('delete Scene', self)
+                rename_scene.triggered.connect(lambda: self._rename_scene(index))
+                delete_scene.triggered.connect(lambda: self._remove_scene(index))
+                menu.addAction(rename_scene)
+                menu.addAction(delete_scene)
+                menu.popup(QtGui.QCursor.pos())
+                break
+
+    def _add_scene(self) -> None:
+        """
```

Comment:
Should we continue to separate these scenes from the one defined in the show file? Appending an optional default value configuration per universe per scene, might come in handy later, as filters won't change universe values, they're not configured for.

This way one could define constant configurations without having to define a plenitude of constants, directly linked to an output filter. Of course one would then be required to send a universe update message right after a scene switch message and would need to highlight the channels in this view that would be overwritten by filters (and maybe vice versa in the filter editor) but it could be a really neat feature and we wouldn't be required to maintain two separate file formats.